thrust/thrust/system/cuda/detail/core/agent_launcher.h

```diff
@@ -36,7 +36,7 @@
 #  pragma system_header
 #endif // no system header
 
-#if _CCCL_HAS_CUDA_COMPILER()
+#if _CCCL_CUDA_COMPILATION()
```

Comment:
I observe that this is not changing the comment at the end of the block, please adopt